### ISSUE TITLE
Is webui.js needed when webui_show() is for type WEBUI_SHOW_URL?

### ISSUE THREAD

**Issue by sensiblearts** [webui-dev/webui#439]
I'm not able to bind events when I do a show like this (Zig):

```Zig
  _ = nwin.show("http://localhost:5984/app");
```
where a node-express server is listening on 5984, serving a static folder starting with `index.html`, which contains `<script src="webui.js"></script>`.

And I noticed that I'm getting 404 on `webui.js`.

Is this intended? (I.e., no need for webui backend when you're serving it from elsewhere?) I looked through the webui C code but I can't tell.

If I substitute a dummy page, it works properly, and event binding works:
```Zig
    const html = "<html><head><script src='webui.js'></script></head> Hello World ! </html>";
     _ = nwin.show(html);
```

In short, is there a way to detect ` WEBUI_EVENT_DISCONNECTED` when show()ing type `WEBUI_SHOW_URL`? I need to detect this event to shut down my child node-express process.

**Comment by sensiblearts** [comment]
Thank you. It was staring me in the face.

Works like a charm:

(Pardon my Zig code -- still learning.)

```Zig
const webui = @import("webui");
const std = @import("std");

const argv = [_][]const u8{ "node", "./index.js" };
var nodeServerProc = std.process.Child.init(&argv, std.heap.page_allocator);

pub fn main() !void {
    // NodeJS backend:
    var dir = try std.fs.cwd().openDir("./src/pdb_nodejs/dist", .{});
    defer dir.close();
    try dir.setAsCwd();
    // REMOVE THESE IF YOU ACTUALLY WANT TO INHERIT THE STREAMS.
    nodeServerProc.stdin_behavior = .Ignore;
    nodeServerProc.stdout_behavior = .Ignore;
    nodeServerProc.stderr_behavior = .Ignore;

    try nodeServerProc.spawn();

    std.debug.print("Spawned process PID={}\n", .{nodeServerProc.id});

    // WebUI frontend:

    var nwin = webui.newWindow();

    _ = nwin.bind("", handleEvent);

    const path = "./src/pdb_nodejs/dist/spa";
    _ = webui.setDefaultRootFolder(path);

    _ = webui.setPort(&nwin, 5983);

    _ = nwin.show("http://localhost:5984/app"); // served by node-express

    webui.wait();

    webui.clean();
}

fn handleEvent(e: webui.Event) void {
    // const stdout = std.io.getStdOut().writer();
    // stdout.print("Event {x}\n", .{@intFromEnum(e.event_type)}) catch {};
    std.debug.print("event {x}\n", .{@intFromEnum(e.event_type)});
    if (@intFromEnum(e.event_type) == 0) {
        if (nodeServerProc.kill()) |_| {
            std.debug.print("Killing node server\n", .{});
        } else |_| {
            std.debug.print("NOT Killing node server\n", .{});
        }
    }
    {}
}

```
